### DISCUSSION TITLE
Сохранение вида оглавления

### DISCUSSION THREAD

**Discussion by furtivite** [doka-guide/platform#336] [Идеи]
У нас есть два вида оглавления:

1. По категориям (по-умолчанию)
![Снимок экрана 2021-09-09 114831](https://user-images.githubusercontent.com/50330458/132654422-78220ed7-4e6f-40ec-9942-efd65be337c5.png)

2. По алфавиту
![Снимок экрана 2021-09-09 114849](https://user-images.githubusercontent.com/50330458/132654462-7b251b26-e3c5-4f9c-8a30-6073405e5aa3.png)

Когда пользователь выбирает вариант (по алфавиту), переходит в любую статью, а потом возвращается снова в оглавление, он снова видит вариант "по категориям". Возможности сохранить вариант отображения оглавления нет. 

Что думаете, чтобы сохранять вариант, который выбрал пользователь?

**Comment by nlopin** [comment]
на время сессии кажется очень разумным решением

**Reply by furtivite** [reply]
Можно сохранять вместе со значением о смене темы.

**Reply by pepelsbey** [reply]
Лайк про сессию, но не больше: мы решили, что группировка по категориям основная, а алфавитная — опция. Не хочется, чтобы группировка могла потеряться, если пришёл через неделю и уже забыл, что она включается.
